docs/aggregation.md

```diff
@@ -1,12 +1,34 @@
-# qgrid 
+# qgrid
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows='data'>
+	<q-grid:columns>
+		<q-grid:column key="Name" title="Name" aggregation="count"></q-grid:column>
```

Comment:
so if aggregation is set for the column we show value in the footer for that column?